workers/unity/Assets/Playground/Editor/SnapshotGenerator/SnapshotGenerator.cs

```diff
@@ -84,14 +83,15 @@ private static void AddCubeGrid(Snapshot snapshot, int cubeCount)
                         return;
                     }
 
+                    var location = new Vector3(x, 1, z);
                     var positionSnapshot = new Position.Snapshot
                     {
-                        Coords = new Coordinates(x, 1, z)
+                        Coords = TransformUtils.ToCoordinates(location)
                     };
                     var transformSnapshot = new TransformInternal.Snapshot
                     {
-                        Location = new Location(x, 1, z),
-                        Rotation = new Quaternion(1, 0, 0, 0),
+                        Location = TransformUtils.ToFixedPointVector3(location),
```

Comment:
Creation of an TransformInternal Snapshot needs a helper utility.
The user should not have to deal with compression.